ceph.spec.in

```diff
@@ -650,6 +650,23 @@ ceph-osd is the object storage daemon for the Ceph distributed file
 system.  It is responsible for storing objects on a local file system
 and providing access to them over the network.
 
+%if 0%{with seastar}
+%package crimson-osd
+Summary:	Ceph Object Storage Daemon
+%if 0%{?suse_version}
+Group:		System/Filesystems
+%endif
+Requires:	ceph-base = %{_epoch_prefix}%{version}-%{release}
+Requires:	ceph-osd = %{_epoch_prefix}%{version}-%{release}
+Requires:	lvm2
+Requires:	sudo
+Requires: libstoragemgmt
```

Comment:
This seems to be misaligned -- I am guessing a mix of spaces and tabs?

Reply:
i removed all Requires but "ceph-osd" of "crimson-osd". since ceph-osd has all the Requires already.